### ISSUE TITLE
Pneumobilia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10054]
**Preferred term label:**
Pneumobilia

**Synonyms**
Air in the biliary system

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C1739097
https://uts.nlm.nih.gov/uts/umls/concept/C2227741

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bouveret syndrome (bilioduodenal fistula)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000454
- label: Pneumobilia
- parents: [Abnormal biliary tract morphology (HP:0012440)](https://hpo.jax.org/app/browse/term/HP:0012440)
- pmid: PMID:16567248
- def: Accumulation of gas in the biliary tree.
- comment: Pneumobilia, or air within the biliary tree of the liver, suggests an abnormal communication between the biliary tract and the intestines, or infection by gas-forming bacteria. Pneumobilia usually can be distinguished from air in the portal venous system by its appearance on computed tomography (CT) scan.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10054
- synonyms: Air in the biliary system(NONE)|Aerobilia(NONE)
